helper.py

```diff
@@ -34,7 +34,7 @@ def load_reader(
     pse_sample_size: int = 64,
     min_area_to_ignore: int = 1000,
     train_or_test: str = "train",
-    alignment: str = "1to2"
+    alignment: str = "1to2",
```

Comment:
I've seen people put commas in the end like this, but I didn't know why.  Can you explain why they do it?

Reply:
It's the default style of my formatter: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas